docs/advanced-usage/multi-domain.mdx

```diff
@@ -0,0 +1,192 @@
+title: Multi domain setup
+description: Learn how to achieve a multi domain setup in order to share authentication and session management across domains.
+---
+
+import { Callout, Steps} from "nextra-theme-docs";
+import { Images } from "@/components/Images";
+import { CodeBlockTabs, Tabs, Tab } from "@/components/CodeBlockTabs";
+
+# Multi domain setup
+
+Clerk supports sharing sessions across multiple domains with a primary and satellite configuration.
+
+Having multiple domains for your instance enables you to have central authentication and session management across all of them.
+
+There is always a primary domain, but there can be many satellite domains.
+
+The primary domain is the source of truth for the user's authentication state. Users must complete the sign-in flow on this domain, either using Clerk’s `<SignIn/>` component or our `useSignIn()` hook.
+
+As for satellite domains, When users initially request one, they are transparently redirected through the primary domain to determine the authentication state. If users want to sign-in, they must be redirected to a sign-in flow hosted on the primary domain, then redirected back.
+
+## How to set up a multi domain project?
+
+<Steps>
+
+### Create your application and install Clerk
+Build your application just like you would any other application with Clerk. Currently, multi-domain can be added to any Next.js or Remix application. For other React frameworks, multi-domain is still supported as long as you do not use server rendering or hydration.
+
+Once you create an instance via the dashboard, you will be prompted to choose a domain. We consider this to be the primary domain.
+
+We will be using the domain `primary.dev` for the purposes of this guide.
+
+<Callout type="info">
+Note: For multi-domain, you must configure your sign-in flow to run within your primary application, for example on `/sign-in`.
+</Callout>
+
+### Configure your primary app
+
+Configuration for this app is the same as for any other app using Clerk without having multiple domains.
+
+<CodeBlockTabs options={["Next.js", "Remix"]}>
+```env copy filename=".env.local"
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_live_...
+CLERK_SECRET_KEY=sk_live_...
+```
+```env filename=".env" copy
+CLERK_PUBLISHABLE_KEY=pk_live_...
+CLERK_SECRET_KEY=sk_live_...
+```
+</CodeBlockTabs>
+
+### Register your satellite domain
+
+Go to the "Domains" tab in the Clerk dashboard and add a new satellite domain of your choice. Then, follow the instructions provided.
+
+We will use the domain `satellite.dev` for the purposes of this guide.
+
+<Images
+src="/docs/images/advance/multi-domain/satellite-domain-add.png"
+width={3024}
+height={1748}
+alt="Clerk Dashboard inside the satellites tab of the domain"
+/>
+
+### Configure your satellite app with environment variables
+
+Now that you have created the satellite application, it's time to configure it as such. You can do so with the environment variables below or via properties (see the next step).
```

Comment:
So do users have to create *two different* applications? One primary, and one satellite? And then the satellite app will have these env variables?

Reply:
Yes. You need to have your code running in two separate applications. The alternative would be to deploy your code to two separate domains, but isn't something running on a different domain a different application?

Code running on foo.com can be the primary and code running on bar.com can be the satellite. The apps can be the same or different. 

@alexisintech How do you think we can best describe this?

Reply:
The docs go from registering a satellite domain to "Now that you have created the satellite application" (but "creating a satellite application) was never talked about.
It feels like a step is missing here. 
What I'm thinking here, is that there should be a step somewhere that says

```
### Create your Satellite application

Now that you have your primary application and your primary domain set up, you need to create your satellite application... (insert more information here... I'm not sure if it's the same process as our [set-up clerk guide](https://clerk.com/docs/quickstarts/setup-clerk) or if it's going to be a different process in the dashboard. I can't really help with writing content here because I can't test things in the dashboard until it's live)

### Configure your Satellite app

Now that you have created the satellite application, it's time to configure it as such. You can do so with the environment variables below or via properties.

#### Configure your Satellite app with environment variables

(insert content)

#### Configure your Satellite app with properties

(insert content)
```